core/templates/pages/about-page/about-page.controller.ts

```diff
@@ -97,17 +85,11 @@ angular.module('oppia').directive('aboutPage', [
 
             WindowRef.nativeWindow.onhashchange = function() {
               const hashChange = window.location.hash.slice(1);
-              if (hashChange === ctrl.TAB_ID_FOUNDATION || (
-                hashChange === 'license')) {
-                activateTab(ctrl.TAB_ID_FOUNDATION);
-                // Ensure page goes to the license section
-                if (hashChange === 'license') {
-                  WindowRef.nativeWindow.location.reload(true);
-                }
-              } else if (hashChange === ctrl.TAB_ID_CREDITS) {
-                activateTab(ctrl.TAB_ID_CREDITS);
-              } else if (hashChange === ctrl.TAB_ID_ABOUT) {
-                activateTab(ctrl.TAB_ID_ABOUT);
+              if (hashChange === ctrl.TAB_ID_LICENSE) {
```

Comment:
Okay onTabClick we are changing the hash + activeTab and once the hash changes we are again changing the tab name, is that's what happening?

Reply:
Yes, I'm not sure why onhashchange event is being implemented, but it may be fixing some bug that it's not so easy to find.

Reply:
Actually I think the event listener should be in the code if you change the hash by hand in the URL, all the logic about change the tab won't happen anymore.

And the onTabClick method needs to be there as well to change the hash automatically.

Reply:
hmmm.. I don't think we should change the tab if someone edits the link, I don't see this behaviour in other pages like exploration editor pages. (Also, I don't think this is expected behaviour in general, like manually adding `#<id-name>` on the URL doesn't change the focus to the element with given ID, [You can try this on wikipedia])

Reply:
>  if you change the hash by hand in the URL, the tab should change

Can you please check whether this is happening right now develop?

Reply:
You can check in other places in Oppia where we have links based on tab, OR you can check this in other websites like Wikipedia.

Reply:
I tried this on https://oppiatestserver.appspot.com/about
1. Go to https://oppiatestserver.appspot.com/about
2. Add #foundation at the end and I don't see ant change in the page

Reply:
This is weird, it worked for me!

Reply:
For some reason, in this link it's working totally fine, but in develop branch is not working as expected.